pytest_reportportal/plugin.py

```diff
@@ -78,6 +78,8 @@ def pytest_sessionstart(session):
                 log_batch_size=int(session.config.getini('rp_log_batch_size')),
                 ignore_errors=bool(session.config.getini('rp_ignore_errors')),
                 custom_launch=session.config.getini('rp_launch_id') or None,
+                rerun=session.config.getini('rp_rerun') or False,
```

Comment:
No need to pass these options to the init_service method. There are being processed in the start_launch only.